modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1268,6 +1268,58 @@ definition {
 		}
 	}
 
+	@description = "LPS-165994 - Verify that the user cannot insert an invalid date on Start / End fields when creating a filter"
+	@priority = "5"
```

Comment:
On the test map and QA validation, this testcase was defined as `@priority = "2"`

Reply:
Changed priority to 2.